Source/HyprMXAdapter.swift

```diff
@@ -67,13 +64,16 @@ final class HyprMXAdapter: PartnerAdapter {
 
         // HyprMX.initialize() uses WKWebView, which must only be used on the main thread
         DispatchQueue.main.async { [self] in
-            // consentStatus will be updated by setGDPR & setCCPA after init
-            HyprMX.initialize(withDistributorId: distributorId,
-              userId: gameID,
-              consentStatus: CONSENT_STATUS_UNKNOWN,
-              ageRestrictedUser: true,  // HyprMX has requested that we simply default to "true"
```

Comment:
Same here, is there any other way we should be setting this info pre-initialization, or does HyprMx just not require this info anymore?

Reply:
According to their changelog there's a new way to set "ageRestrictedUser". We should use it in the `setCOPPA` method

Reply:
Done. (Also did the same for the Android version)